lib/statsd.rb

```diff
@@ -287,10 +292,20 @@ def namespace=(namespace)
   def postfix=(pf)
     case pf
     when nil, false, '' then @postfix = nil
-    else @postfix = "#{@delimiter}#{pf}"
+    else @postfix = ".#{pf}"
     end
   end
 
+  # @attribute [w] postfix
+  #   A value to be appended to the stat name after a '.'. If the value is
+  #   blank then the postfix will be reset to nil (rather than to '.').
+  def tags=(tgs)
+    case tgs
+    when nil, false, '' then @tags = nil
+    else @tags = ",#{tgs}"
+    end
+  end  
```

Comment:
Agreed. I like this better than fitting the `,` in there, as it's a little too hidden. Also it appeared messing with postfix may break things.